api/google-meets/controllers/google-meets.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+const { sanitizeEntity } = require('strapi-utils');
+
+/**
+ * Read the documentation (https://strapi.io/documentation/developer-docs/latest/development/backend-customization.html#core-controllers)
+ * to customize this controller
+ */
+
+
+function userFacingMeets(internalMeet) {
```

Comment:
you could make this function a service

Reply:
I didn't do that because I think this is specific to this module. It transforms the internal representation of meetings into the public representation.

Reply:
by module do you mean the findOne function?

Reply:
No I mean the google-meets controller. If this is exported then other files can import it. But I think other files should work with the internal representation. Controller is the only place we translate internal to public (user) facing representation.